apps/ex_shop/web/models/user.ex

```diff
@@ -6,37 +6,49 @@ defmodule ExShop.User do
     field :email, :string
     field :encrypted_password, :string
     field :password, :string, virtual: true
+    field :password_confirmation, :string, virtual: true
     field :is_admin, :boolean
 
     timestamps
   end
 
-  #@required_fields ~w(name email encrypted_password)
-  #@optional_fields ~w()
+  def changeset(model, params \\ :empty) do
+    model
+    |> cast(params, ~w(), ~w())
+  end
 
   @doc """
-  Creates a register form changeset based on the `model` and `params`.
-
-  If no params are provided, an invalid changeset is returned
-  with no validation performed.
+  Refer Registration changeset
   """
-  def register_admin_form_changeset(model, params \\ :empty) do
-    model
-    |> cast(params, ~w(name email password is_admin), ~w())
-  end
+  def create_changeset(model, params \\ :empty), do: changeset(model, params) |> add_error(:email, "Refer Registration changeset")
```

Comment:
Its added to force and guide that for creation use Registration changeset.
As one might can create one when not aware, and then there will be multiple ways to create user and might not be in sync

Its like a redirection to check registration changeset once, before creating new one